build.sbt

```diff
@@ -1,23 +1,27 @@
-scalaVersion := "2.12.3"
-
-name := "TypeScript importer for Scala.js"
-
-version := "0.1-SNAPSHOT"
+inThisBuild(Def.settings(
+  name := "TypeScript importer for Scala.js",
+  organization := "org.scalajs.tools",
+  version := "0.1-SNAPSHOT",
+  scalaVersion in ThisBuild := "2.12.3",
+  scalacOptions ++= Seq(
+    "-deprecation",
+    "-unchecked",
+    "-feature",
+    "-encoding", "utf8"
+  )
+))
 
 mainClass := Some("org.scalajs.tools.tsimporter.Main")
-
+fork in Test := true
```

Comment:
Why?

Reply:
Is the question regarding reorganizing the file or why to `fork in Test`? I will remove it.

Reply:
It's about why `fork in Test`.

Reply:
The tests are faster now after removing it.